docs/content/preview/migrate/yb-voyager-cli.md

```diff
@@ -265,12 +266,28 @@ Example : `filename1:tablename1,filename2:tablename2[,...]`
 
 Default: '\t' (tab); can be changed to comma(,), pipe(|) or any other character.
 
-### –-has-header
+### --has-header
 
 This argument is to be specified only for CSV file type.
 
 Default: false; change to true if the CSV file contains column names as a header.
 
+**Note**: Boolean flags takes arguments in the format `--flag-name=[true|false]`, and not `--flag-name [true|false]`.
+
+### --file-opts
+
+Comma-separated string options for CSV file format. The options can include the following:
+
+1. `escape_char`: escape character
+
+1. `quote_char`: character used to quote the values
+
+Default: double quotes '"'
```

Comment:
is this working the way you intend?
is there a default escape character?

Reply:
thanks for pointing it out @ddhodge
1. Yes, there is a default escape character, it is also double quotes.
2. Also, We should mention somewhere that escape_char and quote_char are only valid/required for CSV file format(which is the default format)

Reply:
@sanyamsinghal , what are the other file formats that we need to mention apart from CSV. Are others supported?

Reply:
Supported formated are: csv and text